src/modules/rlm_python3/rlm_python3.c

```diff
@@ -41,8 +41,15 @@ RCSID("$Id$")
 #include <link.h>
 #endif
 
+/* Since version 3.8, the "m" suffix is no longer available.
```

Comment:
The comments could follow the other comments.

e.g:

```
/**
  * Since version 3.8, the "m" suffix is no longer available.
  * https://bugs.python.org/issue36707
  */
```

Reply:
Done. I've also updated the comments of the configure.ac to prefix them with `dnl`